File: tests/testdata/codegen/output-name-conflict-pp/go/output-name-conflict.go
package main

import (
	"github.com/khulnasoft/khulnasoft/sdk/v3/go/khulnasoft"
	"github.com/khulnasoft/khulnasoft/sdk/v3/go/khulnasoft/config"
)

func main() {
	khulnasoft.Run(func(ctx *khulnasoft.Context) error {
		cfg := config.New(ctx, "")
		cidrBlock := "Test config variable"
		if param := cfg.Get("cidrBlock"); param != "" {
			cidrBlock = param
		}
		ctx.Export("cidrBlock", cidrBlock)
		return nil
	})
}

File: tests/testdata/codegen/output-name-conflict-pp/nodejs/output-name-conflict.ts
import * as khulnasoft from "@khulnasoft/khulnasoft";

export = async () => {
    const config = new khulnasoft.Config();
    const cidrBlock = config.get("cidrBlock") || "Test config variable";
    return {
        cidrBlock: cidrBlock,
    };
}
